src/python/TaskWorker/Actions/DagmanResubmitter.py

```diff
@@ -121,7 +121,7 @@ def executeInternal(self, *args, **kwargs): #pylint: disable=unused-argument
                     ## files are in the directory resubmit_info in the schedd).
                     for adparam, taskparam in params.iteritems():
                         if taskparam in ad:
-                            schedd.edit(rootConst, adparam, ad[taskparam])
+                            schedd.edit(rootConst, adparam, ad.lookup(taskparam))
                         elif task['resubmit_'+taskparam] != None:
                             schedd.edit(rootConst, adparam, str(task['resubmit_'+taskparam]))
```

Comment:
I think we need to use lookup here as well. There are also line 140 where edit is called, we probably need to change that as well.